pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "databricks-bundle"
-version = "0.9.0b3"
+version = "0.9.0b4"

```

Comment:
version should be changed in separate commit to trigger release of new package version. Please do not do it as a part of PR.